src/Core/src/Handlers/IViewHandler.cs

```diff
@@ -9,7 +20,8 @@ public interface IViewHandler
 		void SetVirtualView(IView view);
 		void UpdateValue(string property);
 		void DisconnectHandler();
-		object? NativeView { get; }
+		NativeView? NativeView { get; }
```

Comment:
This might be an ABI breaking change between net6.0 and net6.0-xxx

Reply:
Changed back for now and put all the gross casting code back